### PR TITLE
Fixed #24550 -- Added migration operation description to sqlmigrate output

### PR DESCRIPTION
**Pull request by MarkusH**
https://code.djangoproject.com/ticket/24550

### CODE REVIEW

**Comment by timgraham** [comment]
Some doc updates are required for the output in the tutorial and GeoDjango tutorial. A sentence in the release notes might be nice.

**Comment by MarkusH** [comment]
Thanks @timgraham . Do you mind proofreading the doc changes.

File: docs/ref/django-admin.txt [hunk 1]
```diff
@@ -984,6 +984,12 @@ By default, the SQL created is for running the migration in the forwards
 direction. Pass ``--backwards`` to generate the SQL for
 unapplying the migration instead.
 
+.. versionchanged:: 1.9
```
Comment (timgraham) [inline docs/ref/django-admin.txt]: If you want to keep this note after `.. versionchanged:: 1.9` is removed, you should reformat it so we don't have to deindent it at that time. I'm not sure it's really important to document this behavior though.

File: docs/releases/1.9.txt [hunk 2]
```diff
@@ -152,6 +152,10 @@ Management Commands
 * The new :djadmin:`sendtestemail` command lets you send a test email to
   easily confirm that email sending through Django is working.
 
+* To increase the readability of the SQL code generated by
+  :djadmin:`sqlmigrate` the SQL code generated for each migration operation is
```
Comment (timgraham) [inline docs/releases/1.9.txt]: comma after sqlmigrate

File: docs/ref/django-admin.txt (hunk 1 above)
Comment (MarkusH) [inline docs/ref/django-admin.txt]: I deliberately indented that note. I don't see it being important past 2.0 or 2.1. It's just regular behavior then. It doesn't break anything (unless the SQL shell somebody is using doesn't support comments :wink:). But keeping it around while people migrate might make sense.

**Comment by MarkusH** [comment]
Merged in c5cc332bf2a0b3ebfa3ad5d26c5b308de5e505be.

**Pull request closed without merging**